mne/evoked.py

```diff
@@ -116,6 +121,11 @@ def save(self, fname):
         ----------
         fname : string
             Name of the file where to save the data.
+            
+        Notes
+        -----
+        To write multiple conditions into a single file, use 
+        :func: `mne.write_evokeds()`.
```

Comment:
this does not render fine. See

https://circle-artifacts.com/gh/mne-tools/mne-python/663/artifacts/0/home/ubuntu/mne-python/doc/_build/html/generated/mne.Evoked.html#mne.Evoked